recipe-box-frontend/src/main/frontend/src/components/RecipeView/RecipeView.tsx

```diff
@@ -0,0 +1,26 @@
+import Header from "../Header/Header";
+import Footer from "../Footer/Footer";
+import {useState} from "react";
+import Categories from "../RecipesList/Categories";
+import "./RecipeView.css";
+import RecipeViewGrid from "./RecipeViewGrid";
+
+
+export default function RecipeView() {
+    const [mealType, setMealtype] = useState("");
+    const [searchQuery, setSearchQuery] = useState("");
+
+    return (
+        <div className="recipe-view-container">
+            <Header/>
```

Comment:
Better create a layout that contains the Header, the Footer and the main element in order to avoid repetition in other page components. More details here:
https://reactrouter.com/en/main/start/tutorial#nested-routes